.lintstagedrc.js

```diff
@@ -0,0 +1,13 @@
+const { CLIEngine } = require('eslint');
+
+const cli = new CLIEngine({});
+
+module.exports = {
+  '**/*.{ts,tsx,js,jsx}': [
+    (changedFiles) =>
+      'yarn format-code ' +
+      changedFiles.filter((file) => !cli.isPathIgnored(file)).join(' '),
+    'yarn test:staged',
+    'prettier --write',
+  ],
+};
```

Comment:
I think it would be nice to have link to the solution in the docs and explanation why we need this

Reply:
To the readme or just add commenting block to this file explaining situation?